ocaml/idl/datamodel.ml

```diff
@@ -9019,7 +9019,7 @@ let pgpu =
         {
           param_type = (Ref _vgpu_type);
           param_name = "value";
-          param_doc = "The VGPU type to enable";
+          param_doc = "The VGPU types to enable";
```

Comment:
I think in this case it's the name of the function that's misleading and not the parameter description, because it's a single `Ref _vgpu_type` we're adding.